assets/airports/eick.json

```diff
@@ -27,658 +27,746 @@
     "defaultMaps": ["Base Map", "Extended Runway Centrelines"],
     "airspace": [
         {
-            "floor": 10,
+            "floor": 50,
             "ceiling": 245,
             "airspace_class": "C",
             "poly": [
```

Comment:
For each airspace polygon of each airport, please use airspace's `labelPositions` property to place the labels at the appropriate position to clearly depict what altitudes are where:
```
"labelPositions": [
    [50.1, 9.25]
],
```

Currently, they're all piled up on top of each other (EICK pictured below as an example):
![image](https://user-images.githubusercontent.com/5103735/96350933-52751900-1086-11eb-99cc-837c42061537.png)

Reply:
Oh man, I didn't know that was a feature 😍